api/src/iot/admin.py

```diff
@@ -32,11 +34,24 @@ def import_xlsx_view(request, message_user, redirect_to):
         try:
             file = request.FILES["selecteer_bestand"]
             workbook = load_workbook(file)
-            errors, created, updated = import_xlsx(workbook)
-            if created:
-                message_user(request, f"{len(created)} sensoren aangemaakt")
-            if updated:
-                message_user(request, f"{len(updated)} sensoren bijgewerkt")
+
+            def action_logger(result):
+                instance, created = result
+                action = 'Aangemaakt' if created else 'Bijgewerkt'
+                LogEntry.objects.log_action(
+                    user_id=request.user.pk,
+                    object_id=instance.pk,
+                    object_repr=str(instance),
+                    content_type_id=ContentType.objects.get_for_model(type(instance)).pk,
```

Comment:
Kunnen we hier niet simpelweg `instance.pk` gebruiken?

Reply:
We hebben de primary key van de ContentType voor het model nodig.